aspnetcore/mvc/razor-pages/index.md

```diff
@@ -205,6 +205,13 @@ The `OnPostDeleteAsync` method:
 * Calls `RedirectToPage` to redirect to the root Index page (`/Index`).
 
 ::: moniker range=">= aspnetcore-2.1"
+
+## Mark Page properties as required
```

Comment:
"Page" in this context ... not a proper noun I think (?)

Since "as" precedes "required," it seems to be calling out the actual attribute name. Therefore, proper noun (?) ... either Required or [Required] (?)